scripts/generate-skippy-api-doc.py

```
@@ -0,0 +1,270 @@
+                    f'<a id="{function_anchor(function)}"></a>',
+                    f"#### `{function.name}`",
+                    "",
+                    function.brief,
+                    "",
+                    "```cpp",
+                    pretty_declaration(function.declaration),
+                    "```",
+                    "",
+                ]
+            )
+        lines.extend([f'<a class="skippy-api-backlink" href="#skippy-function-index">↩ Back to function index</a>', ""])
```

Ruff (0.16.1)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)